core/adapters/amp/amp_api_client.py

```
@@ -0,0 +1,426 @@
+    status: Dict[str, Any] = field(default_factory=dict)
+    messages: List[str] = field(default_factory=list)
+
+    @classmethod
+    def from_dict(cls, data: Dict[str, Any]) -> "UpdateResponse":
+        """Create UpdateResponse from API response dict."""
+        console_entries = []
+        for entry in data.get("ConsoleEntries", []):
+            try:
+                console_entries.append(ConsoleEntry.from_dict(entry))
+            except Exception:
+                pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)